VMBackup/main/ExtensionErrorCodeHelper.py

```diff
@@ -42,7 +42,8 @@ class ExtensionErrorCodeEnum():
     FailedPrepostPluginConfigPermissionError = 317
     FailedPrepostPluginConfigOwnershipError = 318
     FailedGuestAgentInvokedCommandTooLate = 402
-    
+    FailedInputMismatch = 405
```

Comment:
Usually the error code number ranges are grouped based on categories like GuestAgent Error codes, fsfreeze errorcodes, workload failures etc. Let's also make sure the codes are consistent across windows and Linux. Looks like this file has error codes between 11-20 for incorrect input like errors. You can look at adding the new error code within this range.

Reply:
sure will add a code between that